moveit_ros/planning/kinematics_plugin_loader/src/kinematics_plugin_loader.cpp

```diff
@@ -287,31 +288,39 @@ robot_model::SolverAllocatorFn KinematicsPluginLoader::getLoaderFunction(const s
 
       if (default_solver_plugin_.empty())
       {
-        ROS_DEBUG_NAMED(LOGNAME, "Loading settings for kinematics solvers from the ROS param server ...");
+        RCLCPP_DEBUG(LOGGER, "Loading settings for kinematics solvers from the ROS param server ...");
 
         // read data using ROS params
-        ros::NodeHandle nh("~");
-
+        // ros::NodeHandle nh("~");
+        //TODO (anasarrak): Add the appropriate node name for ROS2
+        auto node = rclcpp::Node::make_shared("talker");
```

Comment:
This is a big TODO. Let's do this the right way now

Reply:
@mlautman any suggestion?

Reply:
I like the approach that was used for the controller manager where the node is passed in during initialization as it seems most flexible.